tutorials/evoked/30_eeg_erp.py

```diff
@@ -2,9 +2,9 @@
 """
 .. _tut-erp:
 
-==================================================
-EEG processing and Event Related Potentials (ERPs)
-==================================================
+===============================
+Event Related Potentials (ERPs)
```

Comment:
Please revert this. This tutorial also explains EEG steps and is not focused on ERPs

Reply:
I would like to shorten the title because it looks really ugly now:
![Screen Shot 2022-04-04 at 08 35 10](https://user-images.githubusercontent.com/4377312/161486617-0fbfc9d5-7ec3-4565-8aca-f83890a62bd5.png)

Any ideas? What about "Processing Event-Related Potentials (ERPs)"? I think ERPs are automatically associated with EEG.